gradle-plugins/jaicp-deploy-plugin/src/main/kotlin/com/justai/jaicf/plugins/jaicpdeploy/GradleUtils.kt

```diff
@@ -0,0 +1,20 @@
+package com.justai.jaicf.plugins.jaicpdeploy
```

Comment:
It's quite possible that we'll be adding more plugins for JAICP integration, so I propose using package like
`com.justai.jaicf.plugins.jaicp.build`
Plus, I think this plugin should be `build`. At some point in future could like create `deploy` plugin, which will deploy project from IDEA to JAICP  Cloud directly.